extractorapp/src/deb/resources/etc/georchestra/extractorapp/i18n/extra_keywords_de.properties

```diff
@@ -0,0 +1,5 @@
+success=ERFOLG
```

Comment:
files from this dir should all display the same comments (thos from extra_keywords_en.properties)